src/writeData/subscriptions/components/BrokerFormContent.tsx

```diff
@@ -317,22 +318,25 @@ const BrokerFormContent: FC<Props> = ({
             >
               User
```

Comment:
Is "password" more clear? At the extreme, we could use "username/password" if it fits.

Reply:
Good point - maybe "Basic Auth"?

Reply:
I like it, but question: For consistency, we'd either be leaving the word "auth" off or appending it to the "certificate" option. Does "basic" stand on its own without "auth" behind it? Or is this consistency unnecessary?

Reply:
I think just `Basic` is fine. Lets try it out and if we find it confusing we can always change it down the road